public/scripts/main.js

```diff
@@ -23,18 +27,8 @@ const stage = new Konva.Stage({
 const layer = new Konva.Layer();
 stage.add(layer);
 
-const fightLayer = new Konva.Layer();
-
-// Tooltip for fight
-const fightTooltip = new Tooltip({
-  x: 100,
```

Comment:
These part has been moved to function called fightSceneLoad(), which contains all changes of element about fightscene. So each attack or so, call the function